src/config/babel/babel-react.config.js

```diff
@@ -1,10 +1,5 @@
 module.exports = {
   presets: [
-    [
-      '@babel/preset-react',
-      {
-        runtime: 'automatic',
```

Comment:
it is needed for handling new JSX transform

Reply:
ok